Program.cs

```diff
@@ -32,6 +32,25 @@ public class Program
             };
         }
 
+        internal static int sameplemethod(){
+            var sequence = new int[] { 1, 2, 3, 4, 5, 6, 7, 8, 9 };
+            for (int i = 2; i <= Math.Sqrt(number); i++)
+            {
+                if (number % i == 0)
+                    return false;
+            }
+        }
+
+        internal static int OddManOut(){
+             int[] numbers = { 2, 5, 8, 10, 11, 13, 15, 17, 20 };
+            foreach(int i in sequence){
+                if(i%2 == 1)
+                {
+                    Console.WriteLine($"Squre of odd number {i} is {i*i}");
+                }
+            }
+        }
```

Comment:
**Suggestion:** The `sequence` variable in the `OddManOut` function is not defined. It seems like you wanted to iterate over the `numbers` array. [bug]
```suggestion
        internal static void OddManOut(){
             int[] numbers = { 2, 5, 8, 10, 11, 13, 15, 17, 20 };
            foreach(int i in numbers){
                if(i%2 == 1)
                {
                    Console.WriteLine($"Squre of odd number {i} is {i*i}");
                }
            }
        }
```